index.html

```diff
@@ -122,14 +101,23 @@ <h2>Das Projekt</h2>
               <section>
                 <header>
                   <div class="sticky child">
-                    <h3>Wer bekommt was?</h3>
-                      <p>Bis der Kaffee in eurer Tasse landet, legt er einen langen Weg und eine Reihe an Stationen zurück.</p>
-                    <p>Von uns bekommt ihr darum maximale Transparenz und könnt sehen, wie sich der Preis für unseren Kaffee zusammensetzt. </p>
+                    <p>direkt. fair. persönlich.</p>
                   </div>
                 </header>
                 <div class="content">
                   <p>manoamano - von Hand zu Hand. Was mit einem Non-Profit Projekt begann, der Weitblick-Finanzierung eines neuen Grundschulgebäudes in El Guineo, wird zu All-Profit. Leckerer, duftender Kaffee von den Berghängen El Guineos für uns, faire Löhne für ein Jahr mühevolle Arbeit für die Kleinbauer*innen.</p>
-                  <p>direkt. fair. persönlich. Das macht manoamano einzigartig.</p>
+                  <p>Die erste Kaffeelieferung im Herbst 2019 war ein voller Erfolg. Wir bedanken uns bei unseren vielen Partner*innenn für das Vertrauen und die  Unterstützung. Jetzt arbeiten wir an der Gründung von manoamano als gemeinnützigen Verein - und die nächste Lieferung ist natürlich auch schon in Planung!</p>
+                </div>
+              </section>
+              <section>
+                <header>
+                  <div class="sticky child">
+                    <h3>Wer bekommt was?</h3>
+                    <p>Von uns bekommt ihr darum maximale Transparenz und könnt sehen, wie sich der Preis für unseren Kaffee zusammensetzt. </p>
+                    <p>Bis der Kaffee in eurer Tasse landet, legt er einen langen Weg und eine Reihe an Stationen zurück.</p>
+                  </div>
+                </header>
+                <div class="chld-content">

```

Comment:
Obacht:
```suggestion
                <div class="child-content">

```
der kleine Typo sorgt dafür, dass die Klasse nicht erkannt wird und deswegen der text hier breiter ist als über all anders 😉

Reply:
Upsi, das war mir gar nicht aufgefallen! Danke!